_config/database.yml

```diff
@@ -1,32 +1,7 @@
 ---
-Name: fulltextsearchmysql
-After:
-  - 'databaseconnectors'
+Name: fulltextsearchproxydb
+After: '#fulltextsearchproxydb'
```

Comment:
This is likely causing a circular reference to itself - maybe just remove the `After:` line